graphql/resolvers.go

```diff
@@ -1819,6 +1819,32 @@ func (r *mutationResolver) UnschedulePatchTasks(ctx context.Context, patchID str
 	return &patchID, nil
 }
 
+func (r *mutationResolver) ScheduleFailingBasePatchTasks(ctx context.Context, patchID string) ([]*restModel.APITask, error) {
```

Comment:
Naming wise, I think the main point of this is to restart undispatched base tasks. We happen to only let users do it for the tasks that are failing, but I think that is just a restriction on scheduling undispatched base patch tasks. To me naming it ScheduleUndispatchedBaseTasks feels more readable. We can even add a comment that says something like: 
// we only allow scheduling restarts for undispatched base tasks if the task on the current patch has a failing status. 

This is only my opinion though, so no worries if you disagree.

Reply:
Totally makes sense  Thanks! I just renamed it.